python/src/spark_rapids_ml/classification.py

```diff
@@ -788,3 +862,45 @@ def intercept(self) -> float:
         """
         assert len(self.intercept_) == 1, "multi classes not supported yet"
         return self.intercept_[0]
+
+    def _get_cuml_transform_func(
+        self, dataset: DataFrame, category: str = transform_evaluate.transform
+    ) -> Tuple[_ConstructFunc, _TransformFunc, Optional[_EvaluateFunc],]:
+        coef_ = self.coef_
+        intercept_ = self.intercept_
+        n_cols = self.n_cols
+        dtype = self.dtype
+
+        def _construct_lr() -> CumlT:
+            import numpy as np
+            from cuml.internals.input_utils import input_to_cuml_array
+            from cuml.linear_model.logistic_regression_mg import LogisticRegressionMG
+
+            lr = LogisticRegressionMG(output_type="numpy")
+            lr.n_cols = n_cols
+            lr.dtype = np.dtype(dtype)
+            lr.intercept_ = input_to_cuml_array(
+                np.array(intercept_, order="C").astype(dtype)
+            ).array
+            lr.coef_ = input_to_cuml_array(
+                np.array(coef_, order="C").astype(dtype)
+            ).array
+            return lr
+
+        def _predict(lr: CumlT, pdf: TransformInputType) -> pd.Series:
+            ret = lr.predict(pdf)
+            return pd.Series(ret)
+
+        return _construct_lr, _predict, None
+
+    def _out_schema(self, input_schema: StructType) -> Union[StructType, str]:
```

Comment:
I don't think _out_schema and _transform are needed.  Base classes suffice.   Also prevent probability column from being added during transform.

Reply:
Will address this comment in the next PR, where I will rebase https://github.com/eordentlich/spark-rapids-ml/tree/lr_bench_eo to support probability column.